centaur/src/main/resources/standardTestCases/three_step.test

```diff
@@ -16,5 +16,5 @@ metadata {
   "calls.cgrep.executionStatus": Done
   "calls.ps.runtimeAttributes.docker": "ubuntu:bionic"
   "calls.wc.runtimeAttributes.docker": "ubuntu:latest"
-  "calls.cgrep.runtimeAttributes.docker": "ubuntu:latest"
+  "calls.cgrep.runtimeAttributes.docker": "debian:jessie"
```

Comment:
pure curiosity here, why the change?

Reply:
So I could assert that the `DockerRequirement` on the workflow step was correctly being given precedence over the `DockerRequirement` on the workflow.